pkgs/applications/misc/dstask/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "dstask";
+  version = "v0.18";
+  rev = "d687a0f9aec634a808cfa0df4aecb06ab4f57732";
+
+  src = fetchFromGitHub {
+    owner = "naggie";
+    repo = "dstask";
+    rev = "${version}";
+    sha256 = "16z5zlfj955pzsj0l58835slvpchdaq2vbyx2fjzi6y9xn1z2nd1";
+  };
+
+  vendorSha256 = null;
+
+  buildFlagsArray = [ ''-ldflags=-w -s
+    -X "github.com/naggie/dstask.VERSION=${version}"
+    -X "github.com/naggie/dstask.GIT_COMMIT=${rev}"
+  '' ];
+
+  subPackages = [ "cmd/dstask.go" ];
+
+  meta = with stdenv.lib; {
+    description = "Command line todo list with super-reliable git sync";
+    homepage = "https://github.com/naggie/dstask";
```

Comment:
FYI, you can put `src.meta.homepage` to get the same value :slightly_smiling_face: 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/audio/adlplug/default.nix#L38
https://nixos.org/nixos/packages.html?channel=nixos-20.03&query=adlplug